### ISSUE TITLE
Bug - custom_header broke nuclei and ffuf 

### ISSUE THREAD

**Issue by psyray** [yogeshojha/rengine#950]
@yogeshojha 
This bug #651 is still present although you mentioned it was fixed in this commit https://github.com/yogeshojha/rengine/commit/aa171c20deb8a5bb2c22447af9fa67bea6f892b6

It seems that you only resolve the **httpx** part of this bug

After investigating it seems the split function break the command due to the comma contained in the `-H` args.
When you use `logger.info(subprocess.list2cmdline(command.split()))` you can see that the `"` are escaped
![image](https://github.com/yogeshojha/rengine/assets/1230954/a6b63505-63e1-4364-903f-77675f78b57b)

It could be easily fix using `os.system` like you do for httpx, but it's a quick and dirty fix

Maybe this solution is the best
https://stackoverflow.com/a/14928914

Using the list form of invocation, eg in the SO post
```
command = ["ffmpeg", "-i", "concat:1.ts|2.ts", "-vcodec", "copy", "-acodec", "copy", "temp.mp4"]
output,error  = subprocess.Popen(command)
```

**Comment by psyray** [comment]
Fixed with #951 
Tested & working

**Comment by psyray** [comment]
Fixed in v2
